server/src/main/java/org/elasticsearch/search/aggregations/bucket/range/RangeAggregator.java

```diff
@@ -217,33 +227,53 @@ public boolean equals(Object obj) {
         }
     }
 
-    final ValuesSource.Numeric valuesSource;
+    private final ValuesSource.Numeric valuesSource;
     final DocValueFormat format;
     final Range[] ranges;
+
     final boolean keyed;
-    final InternalRange.Factory rangeFactory;
+    private final InternalRange.Factory rangeFactory;
+    private final double[] maxTo;
 
-    final double[] maxTo;
+    private final String pointField;
+    private final boolean canOptimize;
```

Comment:
I think it'd be easier to read by giving it a more explicit name, e.g. `aggregateUsingPointsIndex` or something along those lines